### PR TITLE
Reformat Live page

### LINKED ISSUE #312: Reformat Live page
Make https://cag.org/live/ more mobile friendly and fix video aspect ratio

### PR DESCRIPTION
**Pull request by ZanyHunter**
Fixes #312 
- Redid iframe formatting
- Darkened and centered content below buttons

Tested and functional under these conditions:
- Stream offline
- Countdown running
- Stream live

### CODE REVIEW

File: live/index.php [hunk 1]
```diff
@@ -88,51 +88,53 @@ function gtag() { dataLayer.push(arguments); }
...
-            <h2>9:00am | 11:00am | 6:00pm ET</h2>
-            <div id="live-iframe">
-                <iframe id="live-stream" class="drop-shadow"
-                    src='https://www.youtube.com/embed/live_stream?channel=<?php echo $channelId ?>' frameborder="0"
+        <h1><span style="color: red; font-weight: bold;">LIVE</span> AT CORNERSTONE</h1>
+        <h2>9:00am | 11:00am | 6:00pm ET</h2>
+        <div id="live-iframe">
+            <div id="resize-wrapper">
+                <iframe class="drop-shadow"
+                    src='https://www.youtube.com/embed/live_stream?channel=UCSJ4gkVC6NrvII8umztf0Ow' frameborder="0"
```
Comment (silentsonata) [inline live/index.php:96]: Is there a reason why this was changed from a dynamic variable fetched by php to static?

File: live/index.php [hunk 2]
```diff
@@ -88,51 +88,53 @@ function gtag() { dataLayer.push(arguments); }
...
-                <a href="/events/" target="_blank" class="button--white">Our Events</a>
-                <a href="/about/" target="_blank" class="button--white">About Us</a>
+            <iframe id="live-chat" frameborder="0" src="https://www.youtube.com/live_chat?v=DWcJFNfaw9c&embed_domain=localhost" style="display: static"></iframe>
+        </div>
+        <div id="stream-offline">
+            <img src="/ref/pages/live_page/offline.png">
+            <h2 id="error-message">Stream Offline</h2>
+            <div id="error-description">
+                <b>Looking for service? Check back at these times:</b>
+                <br>Good Friday Service | 7:00pm
```
Comment (silentsonata) [inline live/index.php:107]: Can you remove the Good Friday service?

File: live/index.php [hunk 3]
```diff
@@ -88,51 +88,53 @@ function gtag() { dataLayer.push(arguments); }
...
-            </div>
-            <div id="countdown">
-                <div id="label-text">Next service starts in</div>
-                <div id="clock"></div>
-            </div>
-            <div id="button-group">
-                <a href="/give/" target="_blank" class="button--white">Give Online</a>
-                <a href="/events/" target="_blank" class="button--white">Our Events</a>
-                <a href="/about/" target="_blank" class="button--white">About Us</a>
+            <iframe id="live-chat" frameborder="0" src="https://www.youtube.com/live_chat?v=DWcJFNfaw9c&embed_domain=localhost" style="display: static"></iframe>
```
Comment (silentsonata) [inline live/index.php:100]: Should the src be empty since it is being fetch and replaced by the PHP script on load?

File: live/index.php (hunk 1 above)
Comment (ZanyHunter) [reply live/index.php]: Not sure how that got there, sorry! Committed reversion.

File: live/index.php (hunk 3 above)
Comment (ZanyHunter) [reply live/index.php]: Hm, guess that was left there from tests. Yeah, the PHP does overwrite it, but it should be empty. Committed fix

File: live/index.php (hunk 2 above)
Comment (ZanyHunter) [reply live/index.php]: Ugh, the result of an error during branch merge. Committed fix

**Comment by ZanyHunter** [comment]
@silentsonata Done! Thanks for catching the errors.

**Comment by silentsonata** [comment]
Great! I will get this in release on Monday. Don't want to merge over the weekend.

**Pull request merged**